### ISSUE TITLE
Add to Atmosphere?

### ISSUE THREAD

**Issue by JulianKingman** [sircharleswatson/Electrometeor#30]
Hi @sircharleswatson, it looks like your repository is probably the most actively developed deployment solution for desktop, have you considered adding it as a package to atmosphere?

**Comment by sircharleswatson** [comment]
@JulianKingman I'm actually working on a CLI and Atmosphere package that will replace the current boilerplate app. :)

**Comment by JulianKingman** [comment]
@sircharleswatson That's great news! Let me know when it's ready, it'll be a nice arrow in the Meteor communities quiver.

**Comment by andrerfneves** [comment]
How much of my current code will break if I've been using your current boilerplate @sircharleswatson? It's a complete rewrite I'm assuming?

**Comment by afang** [comment]
Please update us on the Atmosphere project; otherwise I will create it.

How will the upgrade affect users of the boilerplate app?

**Comment by tenshis** [comment]
Any news on this issue?

**Comment by sjmcdowall** [comment]
@tenshis - have you looked at this package? https://github.com/jrudio/meteor-electron

**Comment by sircharleswatson** [comment]
I don't plan on doing anymore work on this repo at all so other solutions will be better. Especially since this repo hasn't been updated since 1.1 or earlier.

**Comment by sjmcdowall** [comment]
@sircharleswatson -- How about a large RED WARNING or something on the main GitHub page saying this project is noting being worked on -- etc. etc. and maybe pointing them to an ongoing supported one?

**Comment by andrerfneves** [comment]
I don't think that should be @sircharleswatson's job to do so. I do agree the warning on the GH page helps but if you read the readme you'd see this note he left way back in October:

<img width="903" alt="screen shot 2016-05-23 at 4 44 36 pm" src="https://cloud.githubusercontent.com/assets/2574011/15483993/b6a80b1e-2105-11e6-9001-9519c41a046a.png">

You can also check when the last commit was.

<img width="985" alt="screen shot 2016-05-23 at 4 45 28 pm" src="https://cloud.githubusercontent.com/assets/2574011/15484011/c8785b50-2105-11e6-98df-3f370cd7e337.png">

Pointing people to other repos shouldn't be his issue, but it would be beneficial to other devs I guess. Which he just did in his previous response 👍

**Comment by tenshis** [comment]
Thanks guys, I'll take a look at other options.

**Comment by sircharleswatson** [comment]
@sjmcdowall feel free to make a PR to the README.
